cobalt/dom/captions/system_caption_settings.cc

```diff
@@ -629,6 +641,25 @@ const char* SystemCaptionSettings::CaptionOpacityPercentageToString(
   }
 }
 
+bool SystemCaptionSettings::GetCaptionSettings(
+    SbAccessibilityCaptionSettings* caption_settings) {
+#if SB_API_VERSION >= 16
+  auto platform_info_extension =
+      static_cast<const CobaltExtensionAccessibilityAPI*>(
+          SbSystemGetExtension(kCobaltExtensionAccessibilityName));
+  if (platform_info_extension &&
+      strcmp(platform_info_extension->name,
+             kCobaltExtensionAccessibilityName) == 0 &&
+      platform_info_extension->version >= 1) {
+    return platform_info_extension->GetCaptionSettings(caption_settings);
+  } else {
+    return SbAccessibilityGetCaptionSettings(caption_settings);
```

Comment:
It looks to me like this call shouldn't be here, since `SbAccessibilityGetCaptionSettings` isn't defined when Starboard >= 16.

Reply:
Reworked